### PR TITLE
Create HGAlibs2.py

### PR DESCRIPTION
**Pull request by Foster808**
I would like for you to review my code @darkTowerCat

### CODE REVIEW

File: HGAlibs/HGAlibs2.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+""" The Healing Gem of Arcadia Mad Libs Generator """
```
Comment (darkTowerCat) [inline HGAlibs/HGAlibs2.py:1]: When using comments, use a `#`, if you are not using the string.

File: HGAlibs/HGAlibs2.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+word1 = input("Enter a word: ")
+pluralNoun = input("Enter a plural noun: ")
+word2 = input("Enter a second word: ")
+place = input("Enter your favorite place: ")
+adjective = input("Enter an adjective: ")
+word3 = input("Enter a third word: ")
+
+print ("----------YOUR HEALING GEM OF ARCADIA LIBS---------")
```
Comment (darkTowerCat) [inline HGAlibs/HGAlibs2.py:10]: functions should not have a space before the first `(`

File: HGAlibs/HGAlibs2.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+print ("The healing gem and the",word1,"went along with", pluralNoun)
```
Comment (darkTowerCat) [inline HGAlibs/HGAlibs2.py:12]: This works, however for some reason I see `print(f"The healing gem and the {word1} went along with {pluralNoun}")` being used now.

File: HGAlibs/HGAlibs2.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+print ("You may think that The Healing Gem of Arcadia is",word3,",")
+
+print ("And you might be correct. But if you listen deeply you'll really hear its magic.")
+
+print ("------------------------------------------")
+
+print ("------------THANKS FOR PLAYING!!!---------")
+
+print ("------------------------------------------")
+
```
Comment (darkTowerCat) [inline HGAlibs/HGAlibs2.py:31]: Would look into a formatter. black is very popular: https://black.readthedocs.io/en/stable/getting_started.html

**Review by darkTowerCat — APPROVED** [review]
This is really good!

I apologize if the feedback sounds harsh. It always does for me when people review my code, however I want it to sound more friendly and be viewed as constructive feedback instead of trying to sound hurtful. We can also set up a meeting to go over this more in depth, or maybe pair program for a bit if you are interested.

File: HGAlibs/HGAlibs2.py (hunk 3 above)
Comment (darkTowerCat) [reply HGAlibs/HGAlibs2.py:12]: Another option `print("The healing gem and the " + word1 + " went along with " + pluralNoun)`

File: HGAlibs/HGAlibs2.py (hunk 1 above)
Comment (Foster808) [reply HGAlibs/HGAlibs2.py:1]: Aha! Got it.

**Pull request merged**